pkg/apply/processor/install.go

```diff
@@ -117,7 +100,28 @@ func (c *InstallProcessor) SyncStatusAndCheck(cluster *v2.Cluster) error {
 	return nil
 }
 
+func (c *InstallProcessor) ConfirmOverrideApps(cluster *v2.Cluster) error {
+	logger.Info("Executing ConfirmOverrideApps Pipeline in InstallProcessor")
+
+	if ForceOverride || len(c.imagesToOverride) == 0 {
+		return nil
+	}
+
+	prompt := fmt.Sprintf("are you sure to override these following apps? \n%s\t", strings.Join(c.imagesToOverride, "\n"))
+	cancel := "you have canceled to override these apps!"
+	pass, err := confirm.Confirm(prompt, cancel)
+	if err != nil {
+		return err
+	}
+	if !pass {
+		return errors.New(cancel)
```

Comment:
should ignore cancel error?

Reply:
Only moved code place. Do you mean we should ignore appErr if it's cancel?

Reply:
> Only moved code place. Do you mean we should ignore appErr if it's cancel?

yes, cancel is not an unexpected error.

Reply:
ok, add a check at return appErr.